recipes/primp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "primp" %}
+{% set version = "0.9.2" %}
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/primp-{{ version }}.tar.gz
+  sha256: 5b95666c25b9107eab3c05a89cb7b1748d5122e57c57b25bfc3249d525c45300
+
+build:
+  ##noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip:
+    - match(python, "<{{ python_min }}")
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('rust') }}
+    - python >={{ python_min }}
+    - pip
+    #- make
+    #- perl
```

Comment:
Why aren't they needed? Please remove the comment if really not needed.

Reply:
They were part of test. They are not needed. Dropping them.

Reply:
Removed.